### PR TITLE
package reorg: remove pkg connection, rename ConnetionSet to Endpoint…

### PR DESCRIPTION
**Pull request by adisos**
…sTrafficSet

issue #68

### CODE REVIEW

**Comment by zivnevo** [comment]
README.md should be updated to reflect these changes

File: pkg/netset/transportset.go
Comment (zivnevo) [inline pkg/netset/transportset.go:1]: comment in line 17 is no longer relevant

File: pkg/netset/transportset.go [hunk 1]
```diff
@@ -28,13 +28,40 @@ func NewTCPorUDPTransport(protocol netp.ProtocolString, srcMinP, srcMaxP, dstMin
...
+func AllTCPorUDPTransport(protocol netp.ProtocolString) *TransportSet {
+	return NewTCPorUDPTransport(protocol, netp.MinPort, netp.MaxPort, netp.MinPort, netp.MaxPort)
+}
+
+func AllICMPTransport() *TransportSet {
+	return AllOrNothingTransport(false, true)
+}
+
+// AllTCPSetTransport returns a set of connections containing the TCP protocol with all its possible ports
+func AllTCPSetTransport() *TransportSet {
```
Comment (zivnevo) [inline pkg/netset/transportset.go:56]: Suggest renaming to `AllTCPTransport()`

File: pkg/netset/transportset.go [hunk 2]
```diff
@@ -28,13 +28,40 @@ func NewTCPorUDPTransport(protocol netp.ProtocolString, srcMinP, srcMaxP, dstMin
...
+	return AllTCPorUDPTransport(netp.ProtocolStringTCP)
+}
+
+// AllUDPSetTransport returns a set of connections containing the UDP protocol with all its possible ports
+func AllUDPSetTransport() *TransportSet {
```
Comment (zivnevo) [inline pkg/netset/transportset.go:61]: Suggest renaming to `AllUDPTransport()`

File: pkg/netset/transportset.go [hunk 3]
```diff
@@ -55,6 +82,10 @@ func AllTransportSet() *TransportSet {
 	return AllOrNothingTransport(true, true)
 }
 
+func EmptyTransportSet() *TransportSet {
```
Comment (zivnevo) [inline pkg/netset/transportset.go:85]: Suggest renaming to `NoTransports`.
Also, rename `AllTransportSet()` to `AllTransports()`

File: pkg/netset/trafficset.go [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+// IsSubset returns true if c is subset of other
+func (c *EndpointsTrafficSet) IsSubset(other *EndpointsTrafficSet) bool {
+	return c.props.IsSubset(other.props)
+}
+
+// EndpointsTrafficSetFrom returns a new EndpointsTrafficSet object from input src, dst IP-ranges sets ands
+// TransportSet connections
+func EndpointsTrafficSetFrom(src, dst *IPBlock, conn *TransportSet) *EndpointsTrafficSet {
```
Comment (zivnevo) [inline pkg/netset/trafficset.go:81]: This should be `NewEndpointsTrafficSet`.
The current `NewEndpointsTrafficSet` should be `EmptyEndpointsTrafficSet`

File: pkg/netset/netset.go [hunk 5]
```diff
@@ -4,11 +4,11 @@ Copyright 2023- IBM Inc. All Rights Reserved.
...
+// Package netset implements types for network connection sets objects and operations.
 // Types defined in this package:
 // IPBlock - captures a set of IP ranges
 // TCPUDPSet - captures sets of protocols (within TCP,UDP only) and ports (source and destination)
-// ICMPSet - captures sets of types,codes for ICMP protocol
-// TransportSet - captures connection-sets for protocols from {TCP, UDP, ICMP}
-// ConnectionSet - captures a set of connections for tuples of (src IP range, dst IP range, TransportSet)
+// ICMPSet - captures sets of type,code values for ICMP protocol
+// TransportSet - captures union of elements from TCPUDPSet, ICMPSet
+// EndpointsTrafficSet - captures a set of traffic attribute for tuples of (source IP range, desination IP range, TransportSet)
```
Comment (zivnevo) [inline pkg/netset/netset.go:13]:
```suggestion
// EndpointsTrafficSet - captures a set of traffic attribute for tuples of (source IP range, destination IP range, TransportSet)
```

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/netset/transportset.go
Comment (adisos) [reply pkg/netset/transportset.go:1]: fixed

File: pkg/netset/transportset.go (hunk 1 above)
Comment (adisos) [reply pkg/netset/transportset.go]: fixed

File: pkg/netset/transportset.go (hunk 2 above)
Comment (adisos) [reply pkg/netset/transportset.go]: fixed

File: pkg/netset/transportset.go (hunk 3 above)
Comment (adisos) [reply pkg/netset/transportset.go]: fixed

File: pkg/netset/trafficset.go (hunk 4 above)
Comment (adisos) [reply pkg/netset/trafficset.go]: fixed

File: pkg/netset/netset.go (hunk 5 above)
Comment (adisos) [reply pkg/netset/netset.go]: fixed

**Review by zivnevo — APPROVED** [review]

**Pull request merged**